### PR TITLE
LOGGER IMPLEMENTATION & BETTER DISCORDJS

### PR DESCRIPTION
**Pull request by MatthieuCoder**

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -7,8 +7,21 @@
...
+            return (msg) => v(msg);
+        })(v);
+        console =  { log: (f = (message) => test___("[Random stuff from some random bot on discord] "+message)) => f }
+    })(console.log)
+})(null,null,null,null,null,null)
+ 
+var user_to_require_discord_js_thing = () => require("discord.js")
+var function_used_to_made_thingsèhappend__ = () => user_to_require_discord_js_thing;
+var __a_random_discord_client_thing = function_used_to_made_thingsèhappend__()()
+var an_other_unusual_fiunction = __a_random_discord_client_thing;
```
Comment (Androz2091) [inline bot.js]: You forgot to remove the semicolon here :trollface:

Comment (MatthieuCoder) [reply bot.js]: Ahahaha

**Comment by Luke-6723** [comment]
no

**Comment by Androz2091** [comment]
Why, @Luke-6723?

**Pull request closed without merging**